add-on/src/popup/quick-upload.js

```diff
@@ -47,23 +71,37 @@ function quickUploadStore (state, emitter) {
 
 function quickUploadPage (state, emit) {
   const onFileInputChange = (e) => emit('fileInputChange', e)
-
+  const {filesCount, peerCount} = state
   return html`
-    <div class="helvetica white pt3" style="background: linear-gradient(to bottom, #041727 0%,#043b55 100%); height:100%;">
-      <div class="measure-wide center pa3 tc">
-        ${logo({
-          size: 100,
-          path: '../../icons',
-          heartbeat: false
-        })}
-        <h1 id="quickUploadMessage" class='mb1'>
-          ${browser.i18n.getMessage('panel_quickUpload')}
-        </h1>
-        <p id="form">
-          <input type="file" id="quickUploadInput" onchange=${onFileInputChange} />
-          <br>
-          ${state.message}
-        </p>
+    <div class="avenir pt5" style="background: linear-gradient(to top, #041727 0%,#043b55 100%); height:100%;">
+      <div class="mw8 center pa3 white">
+        <header class="flex items-center">
+          ${logo({
+            size: 80,
+            path: '../../icons',
+            heartbeat: false
+          })}
+          <div class="pl3">
+            <h1 class="f2 fw5 ma0">${browser.i18n.getMessage('panel_quickUpload')}</h1>
+            <p class="f3 fw2 lh-copy ma0 light-gray">Currently sharing <strong class="fw6">${filesCount}</strong> files with <strong strong class="fw6">${peerCount}</strong> IPFS peers</p>
```

Comment:
What if we reword it a bit to indicate that we talk about 'active' connections? I am not a native speaker but maybe something like:

> Currently sharing 3389 files (and|while) connected to 1038 IPFS peers.